site/recipes/recipe_adaptable_filter_range.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Adaptable filter behavior
+title: Changing a Filter's Range
 ---
```

Comment:
Please use sentence style capitalization as mentioned in [QUARKS-104](https://issues.apache.org/jira/browse/QUARKS-104): "Changing a filter's range"